router/middleware/repo/repo_test.go

```diff
@@ -147,7 +147,7 @@ func TestRepo_Establish_NoRepoParameter(t *testing.T) {
 	// run test
 	engine.ServeHTTP(context.Writer, context.Request)
 
-	if resp.Code != http.StatusBadRequest {
+	if resp.Code == http.StatusOK {
```

Comment:
What is this change doing? We aren't using the `repo` middleware anymore, right?